bin.js

```diff
@@ -53,7 +53,12 @@ if (argv.help) {
 
 argv.source = argv.source || argv._[0] || '.'
 argv.build = argv.build || 'build'
-argv.layout = argv.layout || 'layout.html'
+argv.layout = argv.layout
```

Comment:
np I already removed it :)

btw you're a collaborator on this project on github and npm: https://www.npmjs.com/package/sitedown/access